frontend/public/components/masthead.jsx

```diff
@@ -38,11 +38,21 @@ export const Masthead = React.memo(({ isMastheadStacked, isNavOpen, onNavToggle
           </PageToggleButton>
         </MastheadToggle>
         <MastheadBrand data-codemods>
-          <MastheadLogo data-codemods component="a" {...logoProps}>
+          <MastheadLogo
+            data-codemods
```

Comment:
should these `data-codemods` still exist?

Reply:
I think they need to be removed, though I can't remember if the codemods do that themselves or if there's another command you need to run. Will check

Reply:
Looks like a rebase removed the codemods tags.